<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>1036</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>249</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>249</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请求调页存储管理方式模拟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="singlestep_bott">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(255,255,255); 
    border: 2px solid #cccccc;
    border-radius: 10px;
    color: #000000;
    padding: 5px 10px;
            }

QPushButton:hover {
    background-color: rgb(124, 203, 86); 
            }
            
QPushButton:pressed {
    background-color: rgb(124, 203, 86); 
            }</string>
   </property>
   <property name="text">
    <string>Single Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_bott">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(255,255,255); 
    border: 2px solid #cccccc;
    border-radius: 10px;
    color: #000000;
    padding: 5px 10px;
            }

QPushButton:hover {
    background-color: rgb(124, 203, 86); 
            }
            
QPushButton:pressed {
    background-color: rgb(124, 203, 86); 
            }</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_bott">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(255,255,255); 
    border: 2px solid #cccccc;
    border-radius: 10px;
    color: #000000;
    padding: 5px 10px;
            }

QPushButton:hover {
    background-color: rgb(124, 203, 86); 
            }
            
QPushButton:pressed {
    background-color: rgb(124, 203, 86); 
            }</string>
   </property>
   <property name="text">
    <string>Auto Execution</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>100</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	text-align: center;
    border-radius: 10px;
    border: 2px solid #cccccc;
    padding: 2px;
}

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_unit">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>条指令/秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>691</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    border: 2px solid black; 
}
QHeaderView::section {
    border-bottom: 2px solid black; 
}




</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>指令序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>逻辑页号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>内存块快号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>物理地址(内存)</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>690</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    border: 2px solid black; 
}
QHeaderView::section {
    border-bottom: 2px solid black; 
}




</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>调入逻辑页页号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>调入内存块块号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>调出逻辑页页号</string>
    </property>
   </column>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>132</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FIFO</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_choice">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>置换算法选择 ：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>200</y>
     <width>132</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LRU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已执行条数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前缺页数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前缺页率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentpro">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>620</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_executed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentnum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>620</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memory">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>内存</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>逻辑页号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>内存编号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起始物理地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memory4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memory3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memory2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memory1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memorystart4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>500</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memorystart1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memorystart3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_memorystart2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_logicnum1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_logicnum4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_logicnum2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_logicnum3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid grey;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid grey;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid grey;</string>
   </property>
  </widget>
  <zorder>widget_3</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>label_main</zorder>
  <zorder>singlestep_bott</zorder>
  <zorder>reset_bott</zorder>
  <zorder>auto_bott</zorder>
  <zorder>input</zorder>
  <zorder>label_unit</zorder>
  <zorder>tableWidget</zorder>
  <zorder>tableWidget_2</zorder>
  <zorder>radioButton</zorder>
  <zorder>label_choice</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_currentpro</zorder>
  <zorder>label_executed</zorder>
  <zorder>label_currentnum</zorder>
  <zorder>label_memory</zorder>
  <zorder>label_3</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_memory4</zorder>
  <zorder>label_memory3</zorder>
  <zorder>label_memory2</zorder>
  <zorder>label_memory1</zorder>
  <zorder>label_memorystart4</zorder>
  <zorder>label_memorystart1</zorder>
  <zorder>label_memorystart3</zorder>
  <zorder>label_memorystart2</zorder>
  <zorder>label_logicnum1</zorder>
  <zorder>label_logicnum4</zorder>
  <zorder>label_logicnum2</zorder>
  <zorder>label_logicnum3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
